### PR TITLE
Finish static pages

### PR DESCRIPTION
**Pull request by trinhducduy**

### CODE REVIEW

File: app/assets/stylesheets/layouts.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto:300,400,700);
+$white: #FFFFFF;
```
Comment (ThuBM) [inline app/assets/stylesheets/layouts.scss]: add new line before

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title
+    base_title = t("application.general.title")
```
Comment (ThuBM) [inline app/helpers/application_helper.rb]: remove ()

File: app/views/layouts/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<footer class="text-center" id="footer">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-12">
+          Copyright &copy; Framgia 2015
```
Comment (ThuBM) [inline app/views/layouts/_footer.html.erb]: I18n

File: app/views/layouts/_footer.html.erb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<footer class="text-center" id="footer">
+    <div class="container">
```
Comment (ThuBM) [inline app/views/layouts/_footer.html.erb]: indent

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -1,14 +1,17 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Base</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <title><%= full_title yield(:title) %></title>
+  <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track" => true %>
```
Comment (ThuBM) [inline app/views/layouts/application.html.erb]: "data-turbolinks-track": true

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -1,14 +1,17 @@
...
+  <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
```
Comment (ThuBM) [inline app/views/layouts/application.html.erb]: "data-turbolinks-track": true

File: app/controllers/static_pages_controller.rb [hunk 7]
```diff
@@ -0,0 +1,4 @@
+class StaticPagesController < ApplicationController
+  def about
```
Comment (ThuBM) [inline app/controllers/static_pages_controller.rb]: static page should have home action

File: app/views/static_pages/about.html.erb [hunk 8]
```diff
@@ -0,0 +1,18 @@
+<% provide :title, t("application.general.about") %>
+<div class="row">
+  <div class="col-md-12 block">
+    <h1 class="text-center">Welcome to Book Review System!</h1>
```
Comment (ThuBM) [inline app/views/static_pages/about.html.erb]: fix I18n in this file

Comment (trinhducduy) [inline app/views/static_pages/about.html.erb]: This is a content file. Should you a separate files for each language(if any) instead I think.

**Comment by ThuBM** [comment]
OK

**Pull request merged**